auth/saml.adoc

```diff
@@ -182,12 +180,12 @@ to be SAML aware and function accordingly.
 Login as admin, then in _Configure->Configuration->Authentication_
 
 * Set mode to External (httpd)
-* Check: _Enable SAML_ - This will enable the SAML login button on the login screen, the redirects
-to the SAML protected page for authentication as well as supporting the SAML logout process.
 * Check: _Enable Single Signon_ - With this option enabled, initial access to the Appliance Administrative UI will
 redirect to the SAML Identity Provider authentication screen.  Note that logouts from the Appliance will
 return the user to the Appliance login screen allowing them to login as admin unless _Disable Local Login_
 is checked below.
+* Check: _Enable SAML_ - This will enable the SAML login button on the login screen, the redirects
+to the SAML protected page for authentication as well as supporting the SAML logout process.
```

Comment:
Recommend changing 'authentication as well as supporting' to 'authentication, and supports' to improve clarity.

Reply:
Thank you @adahms again. Another good suggestions. Will do